generated/provider_dependencies.json

```diff
@@ -104,7 +104,7 @@
     "deps": [
       "apache-airflow-providers-common-sql>=1.20.0",
       "apache-airflow>=2.9.0",
-      "sqlalchemy-drill>=1.1.0,!=1.1.6"
```

Comment:
Don't we want to retain the `>=1.1.0`?

Reply:
Good catch Shahar. Let me add that, dont want to mess that one up even if its obvious :D